src/server/pps/server/api_server.go

```diff
@@ -1387,7 +1383,17 @@ func (a *apiServer) GetLogs(request *pps.GetLogsRequest, apiGetLogsServer pps.AP
 		rcName, containerName string
 		pods                  []v1.Pod
 		err                   error
+		sinceSeconds          *int64
```

Comment:
Why represent this as an integer with units and not a *time.Duration?

Reply:
It was an integer seconds in the original code. But you’re right: it should be a duration. Refactored.